lab-maddy/lib/arithmetic.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+//this is where unit tests will live
+//this module should have add and sub methods
+
+let math = module.exports = {}; //const
+
+math.add = function(a, b){ //numOne, numTwo
+  if((typeof a) && (typeof b) === 'number') {
```

Comment:
This type of comparison doesn't work. You would need to do (typeof a) === 'number' && (typeof b) === 'number'.